findbugs-excludefilter.xml

```diff
@@ -34,7 +34,11 @@
       <Bug pattern="DM_STRING_CTOR" />
       <Bug pattern="RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE" />
       <Bug pattern="RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE" />
+      <Bug pattern="NP_NULL_ON_SOME_PATH" />
       <Bug pattern="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE" />
+      <Bug pattern="NP_NULL_ON_SOME_PATH_MIGHT_BE_INFEASIBLE" />
+      <Bug pattern="NP_NULL_PARAM_DEREF" />
+      <Bug pattern="NP_GUARANTEED_DEREF" />
```

Comment:
There were numerous false-positive findbugs warnings when compiling against the 163.* builds. I'm not sure exactly why this happened, but I did verify that these are false positives, not real issues. Seeing that we were already ignoring similar NP_NULL warnings, this seemed like an appropriate solution.